lib/domain/core/value_objects/app_strings.dart

```diff
@@ -65,6 +65,15 @@ const String resetPasswordDescription = 'Enter your phone number';
 const String sendOTPString = 'Send OTP';
 const String emailOrPhone = 'Email/Phone';
 
+//Verify OTP Page
+const String verifyPhoneNumberTitle =
+    'A one time password has been sent to your phone or email';
```

Comment:
change this to PIN since it is the method we are launching with. Apply for L71 below as well
```suggestion
    'A one time PIN has been sent to your phone or email';
```